lark/exceptions.pyi

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from typing import Dict, Iterable, Callable
+from .tree import Tree
+
+
+class LarkError(Exception):
```

Comment:
Why are the empty classes required? Does mypy gain any information from this?

Reply:
MyPy will check inheritance of the classes. Subclass instance is allow to passed as a superclass instance. ([ref](https://mypy.readthedocs.io/en/stable/kinds_of_types.html#class-types))
Is this exception public? If some one want to catch all Lark exceptions.

Reply:
`LarkError` should be public. So you can either fix that in your patch, or I'll do it in a different patch.

The same is true for most exceptions (most of them are already public)

Reply:
I'll patch it in `lark/__init__.py`.